tests/bluetooth/bsim_bt/bsim_test_audio/src/ccp_test.c

```diff
@@ -202,7 +202,7 @@ static void test_main(void)
 	}
 
 	WAIT_FOR(read_complete);
-	PASS("CCP Passed");
+	PASS("CCP Passed\n");
```

Comment:
With the updated macro, this will not print "PASSED: CCP Passed" - Is that what we want? 

You are also missing the updates for CSIS/CSIP are you not?

Reply:
1) (I guess you mean "now", not "not").  
I do not know what you want :-)  I am OK with this.  (Other modules are similar.)

2) Are any more changes needed for CSIPS?  The macro will print the "PASSED", the string from the test gives more information.  The result will be e.g. "PASSED: All members disconnected".

I am open to change this, I just would like to be able to grep for "pass" and get the results.

Reply:
Regarding updating the FAIL macro: FAILs will print "TESTCASE FAILED", so those are already grep-able.